.github/CONTRIBUTING.txt

```diff
@@ -13,9 +13,16 @@ Here's the long and short of it:
 
       git clone git@github.com:your-username/scikit-image.git
 
-   * Add the upstream repository::
+   * Add the upstream repository:
 
-      git remote add upstream git@github.com:scikit-image/scikit-image.git
+     - If you have set up your `github SSH keys
+       <https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/>`_::
+
+        git clone git@github.com:your-username/scikit-image.git
```

Comment:
Has to be `git remote add upstream git@github.com:scikit-image/scikit-image.git`, hasn't it?